L/LibCURL/build_tarballs.jl

```diff
@@ -70,7 +75,7 @@ dependencies = [
     Dependency("nghttp2_jll"),
     # Note that while we unconditionally list MbedTLS as a dependency,
     # we default to schannel/SecureTransport on Windows/MacOS.
-    Dependency("MbedTLS_jll"),
+    Dependency(Pkg.Types.PackageSpec(name="MbedTLS_jll", version=v"2.24.0")),
```

Comment:
Dummy edit just to trigger rebuild:
```suggestion
    Dependency(Pkg.Types.PackageSpec(; name="MbedTLS_jll", version=v"2.24.0")),
```